x-pack/legacy/plugins/siem/public/components/charts/common.tsx

```diff
@@ -166,3 +141,9 @@ export const getChartWidth = (customWidth?: number, autoSizerWidth?: number): st
   const height = customWidth || autoSizerWidth;
   return height ? `${height}px` : defaultChartWidth;
 };
+
+export const checkIfAllValuesAreZero = (data: unknown): boolean =>
+  Array.isArray(data) &&
+  data.every(series => {
+    return Array.isArray(series.value) && (series.value as ChartData[]).every(({ y }) => y === 0);
```

Comment:
should this be recursive?

Reply:
I'm expecting the data passed in should be `an array of ChartSeriesData`, 
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/public/components/charts/common.tsx#L78-L82

```
export interface ChartSeriesData {
  key: string;
  value: ChartData[] | [] | null;
  color?: string | undefined;
}

```
probably I should change the type of the data from `unknown` to `ChartSeriesData[]`  here to make it clear that it is iterating through a specific structure.

Reply:
Yes I'd be happy with that change, thanks